js/tests/unit/button.js

```diff
@@ -60,6 +60,19 @@ $(function () {
     assert.strictEqual($btn.attr('aria-pressed'), 'true', 'btn aria-pressed state is true')
   })
 
+  QUnit.test('should toggle aria-pressed on buttons with container', function (assert) {
```

Comment:
@mdo is this just a problem of configuration? running the test locally worked just fine with no errors